src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java

```diff
@@ -145,6 +147,15 @@ public void setUp() throws Exception {
         when(headResponse.getStatusCode()).thenReturn(200);
     }
 
+    @After
+    public void tearDown() {
+        try {
+            FileUtils.deleteDirectory(new File(exportDirectory));
```

Comment:
Probably don't need this, files are in the target directory and will get cleaned with `mvn clean`.